lib/cpuset/cpuset.go

```diff
@@ -168,7 +172,12 @@ func Parse(s string) (CPUSet, error) {
 		if err != nil {
 			return New(), err
 		}
+
 		for v := lower; v <= upper; v++ {
+			if v > math.MaxUint16 {
+				return New(), fmt.Errorf(
+					"failed to parse element %s, more than max allowed cores", set)
```

Comment:
Minor nit: not sure if `gofmt` has a specific line length requirement, but this seems to be same length and indentation level as the previous error message, so it could be set as a single line as well.

Reply:
`gofmt` doesn't care, but I try to keep the line lengths reasonable just for readability. But there's not really a good way to break this line without having it still be super long, so I'll make it match the previous one.